pkg/apiserver/controller/cluster/cluster.go

```diff
@@ -69,6 +70,34 @@ type ListClusterRequest struct {
 	ClusterStatus   string   `json:"clusterStatus"`
 }
 
+type ListClusterByLabelRequest struct {
+	ClusterNameList []string `json:"clusterNames"`
+	// 过滤标签，指定节点或Pod标签过滤。默认返回全部
+	labels string `json:"labels"`
+	// 过滤标签的类型，取值是枚举类型，node、pod
+	labelType string `json:"labelType"`
+	// 分页参数，页编号，默认1
+	pageNo int `json:"pageNo"`
+	// 分页参数，页大小，默认500
+	pageSize int `json:"pageSize"`
+}
+
+type ListClusterByLabelResponse struct {
```

Comment:
It might be better to rename `ListClusterByLabelResponse` to `ListClusterResourcesResponse`